src/error.rs

```diff
@@ -135,6 +135,17 @@ pub enum ConvertError<A, S, V> {
     Validity(V),
 }
 
+impl<Src, Dst: ?Sized + Unaligned, S, V> ConvertError<AlignmentError<Src, Dst>, S, V> {
+    // TODO: Bikeshed this name
+    pub fn recall_aligned(self) -> ConvertError<Infallible, S, V> {
```

Comment:
I continue to stand by my extreme fondness for `recall_*` as a naming pattern. :-)

Reply:
I just updated and added `Into` impls that I think supersede this - I think we can just remove it entirely.

Reply:
Discussed offline; removed.